shared/chat/conversation/messages/wrapper/container.js

```diff
@@ -83,8 +83,10 @@ const mergeProps = (stateProps, dispatchProps) => {
   const timestamp =
     stateProps.orangeLineAbove || !previous || oldEnough ? formatTimeForMessages(message.timestamp) : null
   const includeHeader = !previous || !continuingTextBlock || !!timestamp
-  const failureDescription =
-    message.type === 'text' || message.type === 'attachment' ? message.errorReason : null
+  let failureDescription = null
+  if ((message.type === 'text' || message.type === 'attachment') && message.errorReason) {
```

Comment:
This is another case where I think this stuff would be better off using the types we generate from AVDL instead of these strings. Seems easy to mess up.